app/controllers/search.js

```diff
@@ -43,15 +43,17 @@ export default Controller.extend({
       searchPhrase: this.inputPhrase,
       clicked: {
         type: 'article', // currently the only displayed type in the search
-        id: result.id,
+        id: result.wikiId + '_' + result.id,
         title: result.title,
         position: result.position + 1, // +1 since we need to start with 1 instead of 0
         thumbnail: false, // we do not show thumbnails on SRP right now
```

Comment:
I don't see anything that would indicate that it's a cross-wiki article search other than the ID including the wiki ID of possibly another wiki. If that's the only indication how do we differentiate clicks on articles from wiki the user is currently on in both search modes?